packages/terra-infinite-list/src/InfiniteList.jsx

```diff
@@ -234,6 +234,7 @@ class InfiniteList extends React.Component {
     if (!this.contentNode) {
       return;
     }
+    clearTimeout(this.timer);
```

Comment:
@lokesh-0813, could let me know how this callout to cleartimeout will make the scrollheight value not null ? or is this call prevent certain calls which was happening before ?

Reply:
@ShettyAkarsh, scrollHeight is used in handleResize() which is called inside setTimeout() and when clearTimeout() is called, it terminates setTimeout().